### DISCUSSION TITLE
월드에 입장했을때 머리카락이 걸을때 부터 휘날리는 문제

### DISCUSSION THREAD

**Discussion by mhkim0720** [naverz/zepeto-studio-kor#147] [5. Q&A]
안녕하세요 

캐릭터 머리카락이 다른 맵은 달릴때 부터 휘날리는데 제가 작업한 맵은 걸을 때부터 휘날리는 현상이 있는데요 
이건 어디를 확인해봐야 할까요 ? 

확인 부탁드립니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 
문제 해결을 위해, 상황의 범위를 조금 더 좁혀주시면 구체적인 도움을 드릴 수 있도록 하겠습니다.

1. (정상적인) 다른 맵이라 함은, 동일하게 제페토 스크립트를 통해 제작된 월드 컨텐츠 일까요?
2. 정지상태에서는 문제가 없다가, 움직임이 발생하면 속도와 무관하게 (머리카락이 휘날리는) 최대 효과가 발생하는 상황이실까요?

**Reply by mhkim0720** [reply]
안녕하세요. 

더 확인 해보니 

1. 제페토 스크립트를 통해 제작된 월드에서는 긴머리일 경우 걷기 만해도 동일하게 휘날리는 현상이 있는 거 같고요 
단발머리 일 때는 어느 맵 에서는 휘날리고 안 휘날리고 가 차이가 나는 거 같습니다. 

2. 헤어(단발+브릿지) 에 따라 정지 상태 일 때도 머리가 구부러 지거나 하는 현상도 발견 되었습니다.  

확인 부탁 드립니다.
